QuantumExperience/QuasmDriver.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Quantum.Primitive;
+using Microsoft.Quantum.Simulation.Core;
+using Microsoft.Quantum.Simulation.Simulators;
+
+namespace Quasm
+{
+    /// <summary>
+    /// Quick and dirty Simulatorbase
+    /// Just enough to show that it would work
+    /// Please don't put this in production until its fully engineerd.
+    /// This code could eat your cat. So imagine what schrodinger has to say about that one.
+    /// </summary>
+    public abstract class QuasmDriver : SimulatorBase
+    {
+        protected abstract List<Result> RunQuasm(StringBuilder quasm);
+        public abstract int QBitCount { get; }
+       
+        public class QSimH : H
+        {
+            public QSimH(IOperationFactory m) : base(m)
+            {
+            }
+
+            public override Func<Qubit, QVoid> Body
+            {
+                get
+                {
+                    return delegate (Qubit q1)
+                    {
+                        if (q1 == null)
+                        {
+                            return QVoid.Instance;
+                        }
+                        QuasmLog.AppendLine("h q[" + (uint)q1.Id + "];");
```

Comment:
Interpolated strings IMHO would make it a bit easier to read, i.e.: 
`QuasmLog.AppendLine($"h q[{q1.Id}];"); `

Reply:
Would indeed make it more readable.